src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -273,15 +314,17 @@ impl Deref for Node {
 #[derive(Clone,CloneRef,Debug)]
 #[allow(missing_docs)]
 pub struct NodeModel {
-    pub app            : Application,
-    pub display_object : display::object::Instance,
-    pub logger         : Logger,
-    pub main_area      : component::ShapeView<shape::Shape>,
-    pub drag_area      : component::ShapeView<drag_area::Shape>,
-    pub input          : input::Area,
-    pub output         : output::Area,
-    pub visualization  : visualization::Container,
-    pub action_bar     : action_bar::ActionBar,
+    pub app             : Application,
+    pub display_object  : display::object::Instance,
+    pub logger          : Logger,
+    pub main_area       : component::ShapeView<shape::Shape>,
+    pub drag_area       : component::ShapeView<drag_area::Shape>,
+    pub error_indicator : component::ShapeView<error_shape::Shape>,
```

Comment:
why not `error_area`? I believe it is an area, looking at the screenshots, and then it would have consistent name with other areas.

Reply:
I find `indicator` more apt. It has no area properties, instead, it something that can visually indicate that there is an error. It's only states are on/off (with animations) and we don't care about area-based interactions with it.